lighthouse-core/scripts/legacy-javascript/create-polyfill-size-estimation.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-disable no-console */
+
+/**
+ * @fileoverview - Used to generate size estimation data for polyfills in LegacyJavaScript audit.
+ *
+ * Returns a flattened graph of modules found in bundles used for an individual core-js polyfill.
```

Comment:
could we get a tad more explaining the structure of the file itself that it generates?